model/common/src/icon4py/model/common/constants.py

```diff
@@ -79,7 +79,9 @@
 # Implementation constants
 #: default physics to dynamics time step ratio
 # TODO (magdalena) not a constant, this is a default config parameter
-DEFAULT_PHYSICS_DYNAMICS_TIMESTEP_RATIO: Final[float] = 5.0
+DEFAULT_PHYSICS_DYNAMICS_TIMESTEP_RATIO: Final[
+    float
+] = 5.0  # TODO isn't this the inverse i.e. should it be called DEFAULT_DYNAMICS_TO_PHYSICS_TIMESTEP_RATIO
```

Comment:
Agree with the name change.... 

Also: can you remove the `# TODO ` above. I don't know why I put that here. It is the default not the dynamic substep ratio, so it _is_ a constant after all.